### ISSUE TITLE
Failed optimisation of hyperparameter variables

### ISSUE THREAD

**Issue by haroldfinch69** [hyperopt/hyperopt#757]
Hi All,

I've implemented a tuning process and it works 90% of the time however I've discovered a number of tuning runs fail the outputting parameters resulting in a terrible model in terms of the training and validation set.

The failed training precision recall curve is:
![image](https://user-images.githubusercontent.com/73319257/96941930-36e69400-1506-11eb-95c6-9a7ec236de11.png)
The successful curve is:
![image](https://user-images.githubusercontent.com/73319257/96941950-4d8ceb00-1506-11eb-9dfd-632fd5831d2c.png)

In addition the results of the trials look different to a successful optimisation, the failed tuning is:
![image](https://user-images.githubusercontent.com/73319257/96942111-cc822380-1506-11eb-9fc2-4026d1b4e5d2.png)
And the successful tuning is:
![image](https://user-images.githubusercontent.com/73319257/96942016-81681080-1506-11eb-9117-d06da8986e25.png)

These have both been optimised against the F1 score.  I'm just trialling changing the optimisation metric but are there are other tips or strategies I can implement to prevent this happening?

Thanks

**Comment by haroldfinch69** [comment]
comment
